extensions/blocks/membership-button/editor.scss

```diff
@@ -30,4 +30,10 @@
 		border: 1px solid;
 		border-color: var( --color-error );
 	}
+
+	.membership-button__disclaimer {
+		margin-top: 20px;
+		font-style: italic;
+		color: var( --color-gray-400 );
```

Comment:
You'll need to use something darker for accessibility:

https://webaim.org/resources/contrastchecker/?fcolor=7C848B&bcolor=F3F3F4

Reply:
Now that whole link is externallink, grey will not show anymore. I removed that color.